src/v0/analytics.ts

```diff
@@ -202,11 +217,20 @@ export class Analytics {
     })
   }
 
-  getVouchersReports(): Promise<AnalyticsResponse> {
+  getVouchersReports(query?: VoucherOptions): Promise<AnalyticsResponse> {
     return new Promise(async (resolve, reject) => {
```

Comment:
Not sure, but we might need to set a default (undefined), as query is optional

Reply:
added test for both query input and undefined input to make sure this works